src/simplewallet/simplewallet.cpp

```diff
@@ -2319,7 +2412,15 @@ bool simple_wallet::transfer_main(int transfer_type, const std::vector<std::stri
             dust_not_in_fee += ptx_vector[n].dust;
         }
 
+        subaddr_indices.clear();
+        for (auto& ptx : ptx_vector)
+          for (uint32_t i : ptx.construction_data.subaddr_indices)
+            subaddr_indices.insert(i);
         std::stringstream prompt;
+        for (uint32_t i : subaddr_indices)
+          prompt << boost::format(tr("Spending from address index %d\n")) % i;
+        if (subaddr_indices.size() > 1)
+          prompt << tr("WARNING: Outputs of multiple addresses are being used together.\n");
```

Comment:
> Technically, there might be only one subaddress per tx (though fairly unlikely).

What do you mean? I'm confused. This part tells the user which subaddress indices are used as inputs to construct the transaction.